index.js

```diff
@@ -127,7 +127,9 @@ function Linux(instance, end) {
       command = command.join(' ');
       Node.child.exec(command,
         function(error, stdout, stderr) {
-          if (error && /Request dismissed|Command failed/i.test(error)) {
+          if (/No authentication agent found/.test(stderr)) {
```

Comment:
Could you add an `error &&` to the start of the condition? This restricts the check to our expectations, and prevents us from scanning `stderr` (which may be large) unnecessarily.

Alternatively, we could also nest the `if...else` block within a `if (error) {` block.